tests/containers/suma_containers.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test SUMA container images:
+#   The scope of the testing is minimal and covers pulling and running 
+#   container images from the internal registry.suse.de
+#   
+# The container images tested are:
+#   - proxy-httpd
+#   - proxy-salt-broker
+#   - proxy-squid
+#   - proxy-ssh
+#   - proxy-tftpd
+#
+# The testing of proxy-charts is limited to pulling the tar from the registry to
+# and inspect the content to ensure Charts is available. 
+# Install or deployment is not covered in this scenario.
+#   
+# In depth testing of all the images  is already done by the SUMA QE team
+#
+# Maintainer: Maurizio Galli <maurizio.galli@suse.com>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    select_serial_terminal;
+
+    my $image = get_required_var('CONTAINER_IMAGE_TO_TEST');
+
+    if (check_var('CONTAINER_SUMA', 'image')) {
+        my $runtime = get_required_var('CONTAINER_RUNTIME');
+        script_retry("$runtime pull $image", timeout => 300, delay => 60, retry => 3);
```

Comment:
My previous remark that `$runtime` dan be a set of runtimes (e.g. `podman,docker`) somehow got lost and we need to re-implement this.

See e.g. https://duck-norris.qe.suse.de/tests/13835#step/suma_containers/9

Reply:
Weirdly this was working before, @m4u9 could it be that this somehow got overwritten? :thinking:

Reply:
We discussed this in Slack - Please just ping me if you still need help. We're almost there :slightly_smiling_face: